document/core/valid/instructions.rst

```diff
@@ -1458,11 +1460,11 @@ Control Instructions
 :math:`\RETHROW~l`
 ..................
 
-* The label :math:`C.\CLABELS[l]` must be defined in the context.
+* The label type :math:`C.\CLABELS[l]` must be defined in the context.
```

Comment:
Actually, it _is_ the label that must be defined (_with_ a type). So I'd revert this (similarly several occurrences below).

Reply:
Done, for the other occurrences as well.